cypress/TestCases/FireboltCertification/Device.feature

```diff
@@ -51,7 +51,7 @@ Feature: Device
         Then 'Firebolt' platform responds to '1st party app' for 'set device name to kitchen'
         And 'Firebolt' platform does not trigger event for 'onDeviceNameChanged'
 
-    @Device @coreSDK @sdk @transport @notSupported
+    @Device @coreSDK @sdk @transport @requiresPlatformImplementation
```

Comment:
don't remove @notSupported tag for TP test cases. those are still required along with requiresPlatformImplementation tag